cmd/api/main.go

```diff
@@ -490,3 +365,215 @@ func configureTelemetry(
 		return nil
 	}, nil
 }
+
+func restoreBackup(databaseURL string, backupURL string) error {
```

Comment:
Each new builder function tries to follow to general rules:
1) Receive as parameters the most concrete type as possible.
2) Return a `closerModule` function if whatever it built needs to be gracefully closed by main.